src/main/java/org/testng/internal/IConfigInvoker.java

```diff
@@ -25,23 +24,7 @@ void invokeAfterGroupsConfigurations(
       Map<String, String> params,
       Object instance);
 
-  void invokeConfigurations(
-      IClass testClass,
-      ITestNGMethod[] allMethods,
-      XmlSuite suite,
-      Map<String, String> params,
-      Object[] parameterValues,
-      Object instance);
-
-  void invokeConfigurations(
-      IClass testClass,
-      ITestNGMethod currentTestMethod,
-      ITestNGMethod[] allMethods,
-      XmlSuite suite,
-      Map<String, String> params,
-      Object[] parameterValues,
-      Object instance,
-      ITestResult testMethodResult);
+  void invokeConfigurations(ConfigMethodAttributes configMethodAttributes);
```

Comment:
Not a problem for me but no deprecation strategy here?

Reply:
Deprecation strategy is not required here, because it was me who introduced this method in the interface and that too only in the last commit :)